tests/integration_tests/functional/test_drives.py

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
I think it is better if this commit is squashed with the previous one, since it dismisses some changes from there.

Reply:
Thanks. Will do.